scripts/releaseChangelog.mjs

```diff
@@ -263,7 +263,9 @@ ${logChangelogSection(pickersProCommits)}
 
 ${logChangelogSection(chartsCommits)}
 
-### Tree View / \`@mui/x-tree-view@__VERSION__\`
+### Tree View
+
+#### \`@mui/x-tree-view@__VERSION__\`
```

Comment:
Any particular reason for this change? 🤔 
Charts also use such formatting until there is a pro package. 🤔

Reply:
I see two problems with the current structure

- We introduce a structure for some packages and another structure for others
- When we have BCs, we don't have a good place to put them

Reply:
Those are very good points.
I'm not against this change, I just wanted to mention the discrepancy. 🤔 
IMHO, if we change it for `TreeView`, the same should probably apply to `Charts` as well. 🙈

Reply:
@alexfauquette Would you be OK doing the same for charts?
Especially given that both charts and tree view should get a pro package in the coming months

Reply:
Ok for me :+1:

Reply:
If it is agreed upon, shall we update the `Charts` script section as well?
And also apply this change to the existing PR changelog? 🤔

Reply:
I updated the script
I'll update the CHANGELOG

Reply:
Done